.github/styles/Google/WordList.yml

```diff
@@ -55,7 +55,6 @@ swap:
   static IP address: static external IP address
```

Comment:
I found a broken link in https://docs-dev.timescale.com/docs-3769-docs-rfcrename-user-defined-actions-as-jobs-throughout-the-docs/api/3769-docs-rfcrename-user-defined-actions-as-jobs-throughout-the-docs/hypercore/add_columnstore_policy/, can you double check the links please.

Reply:
done